ERCS/erc-7806.md

```diff
@@ -0,0 +1,451 @@
+---
+eip: 7806
+title: Minimal intent-centric EOA smart account
+description: Minimal effort intent-centric interfaces for EOA account abstraction
+author: hellohanchen (@hellohanchen)
+discussions-to: https://ethereum-magicians.org/t/erc-7806-minimal-intent-centric-eoa-smart-account/21565
+status: Draft
+type: Standards Track
+category: ERC
+created: 2024-11-02
+requires: 7702
+---
+
+## Abstract
+
+This proposal defines interfaces to build intent-centric smart accounts. EOA account can delegate contract code to the
+smart account implementation and sign intents to allow solvers (and relayers) to execute on behalf of account owner.
+
+## Motivation
+
+AA (Account abstraction) is a hot topic in blockchain industry because it gives accounts programmability features,
+including but not limited to:
+
+* **BatchExecution**
+* **GasSponsorship**
+* **AccessControl**
```

Comment:
Why are these joined into single words? If they are code (and I don't believe they are), you should put them in backticks (\`). Otherwise, put a space between the words.

Reply:
Updated 👍